### ISSUE TITLE
Display timestamps in local time format e.g. 24h vs. 12h

### ISSUE THREAD

**Issue by Danfro** [holochain-apps/volla-messages#168]
Currently time stamps of messages are given in UTC time. It would be more convenient to have local time displayed.

**Comment by mattyg** [comment]
AC:
- As a user I should see all timestamps in my local timezone
- use a utility library such as date-fns to handle local time zone conversions

**Comment by Danfro** [comment]
No, I don't. And actually its not even the right difference between UTC and my timezone (UTC+1 now with daylight saving having ended).

![grafik](https://github.com/user-attachments/assets/d563c0a1-a3e9-44a8-8fed-0db6120f5598)

**Comment by mattyg** [comment]
Hi @Danfro Yes I understand. I'm just noting the acceptance criteria of this ticket which I'll use to confirm its resolution. Thanks.

**Comment by mattyg** [comment]
Hi @Danfro we're trying to reproduce this issue. I see that in your screenshot it looks like your phone's system time is 22:24, and the message time is 10:23pm (i.e. 22:23 in 24hr time). 

So it looks to me like you're seeing the same time zone in your message as the android system. Is that correct or am I missing something?

Are you wanting to see the time in 24 hour format in the app?

**Comment by Danfro** [comment]
@mattyg I am sorry, you are right. The timezone seems to be fine. But of course it would be nice to have an option to get 24h format if possible. Either via setting or adapted to what is set in the system.

**Comment by mattyg** [comment]
Closing as duplicate of https://github.com/holochain-apps/volla-messages/issues/168
